.travis.yml

```diff
@@ -22,9 +22,12 @@ cache:
 
 before_install:
   - sudo apt-get update
-  - sudo apt-get -y install python3-pip python-dev
-  - sudo pip3 install -U pip setuptools virtualenvwrapper
-  - sudo pip3 install -U coala-bears --pre
```

Comment:
Note the --pre isnt relevant, as it was from before the current release.
The tests fail on all versions with coala 0.7-0.11.